<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 153, 102)

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="record_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255,170,128)</string>
    </property>
    <property name="text">
     <string>REKAM SUARA</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../lenovo/Downloads/mic.svg</normaloff>../../../lenovo/Downloads/mic.svg</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255,170,128)</string>
    </property>
    <property name="text">
     <string>KELUAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../lenovo/Downloads/log-out.svg</normaloff>../../../lenovo/Downloads/log-out.svg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 24pt &quot;Dubai Medium&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>MATEMATIKA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bangun_ruang_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255,170,128)</string>
    </property>
    <property name="text">
     <string>BANGUN RUANG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="luas_bangun_ruang_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255,170,128)</string>
    </property>
    <property name="text">
     <string>LUAS BANGUN RUANG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pecahan_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255,170,128)</string>
    </property>
    <property name="text">
     <string>PECAHAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operasi_pecahan_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255,170,128)</string>
    </property>
    <property name="text">
     <string>OPERASI PADA PECAHAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="perbandingan_btn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
background-color: rgb(255,170,128)</string>
    </property>
    <property name="text">
     <string>PERBANDINGAN</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
